lib/ExceptionTransformer.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 const mapUtils_1 = require("./utils/mapUtils");
+const errorConstants_1 = require("./utils/errorConstants");
```

Comment:
Why do we need to add a number suffix?

Reply:
This is the output file. You can just review the `ts` files